src/assets/scripts/aircraft/AircraftController.js

```diff
@@ -353,6 +352,25 @@ export default class AircraftController {
         return this.aircraft.models[icao];
     }
 
+    /**
+     * Remove the specified aircraft from `AircraftController.aircraft`
+     * @method removeAircraftInstanceModelFromList
+     * @param  {Aircraft} aircraft the aircraft to remove
+     */
+    removeAircraftInstanceModelFromList(aircraft) {
+        this.aircraft.list = _without(this.aircraft.list, aircraft);
+    }
+
+    /**
+     * Remove a flight number from the list stored in `AircraftController.aircraft.callsigns`
+     * @for AircraftController
+     * @method removeCallsignFromList
+     * @param  {string} callsign the flight number to remove
```

Comment:
You are correct. The callsign is the full name, eg `AAL123`. The `AAL` should be considered the "airline" and `123` should be considered the "flight number". There's lots of improperly named things throughout, maybe one day they will all go away. :smile: Glad you called this out though.